sup3r/models/base.py

```diff
@@ -77,6 +76,8 @@ def __init__(self, gen_layers, disc_layers, loss='MeanSquaredError',
             normalization scheme between transfer learning domains.
         name : str | None
             Optional name for the GAN.
+        verbose : bool
```

Comment:
I dont think you need this here. I added these flags to the load() kwargs.

Reply:
Yeah but I don't want the version info logged for every single forward pass chunk

Reply:
Seeing your most recent commit, I get it now. I think we can actually just move this logging statement into the load() method. There's no reason to print versions on init, just when you're loading. Thoughts?

Reply:
Yeah I like that